gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
```

Comment:
I chose to solve it in `safeSpawnSync` to prevent issues for other (or future) commands. It's not entirely clear to me if Windows requires it all the time or just for some commands, so in this way we do not have to duplicate it and prevent any issues.